app/views/pages/derivatives.html.erb

```diff
@@ -0,0 +1,160 @@
+<% provide(:title, 'Derivatives') %>
+
+<% cache do %>
+<div class="container">
+  <h1 class="about_h1">Using the Archives Unleashed Cloud Derivative Files</h1>
+  <p class="about_p">One of the main features of the Archives Unleashed Cloud is the creation of <strong>derivative files</strong>. These are datasets that provide information about your web archives and can be used by researchers in place of the original WARC files.</p>
+  <p class="about_p">Once analytics have been run on a collection, as discussed in the <a href="/documentation">AUK documentation</a>, you will be able to view and access a series of four  derivative files.</p>
+  <div class="card-deck">
```

Comment:
For the images, right now the Gephi and the Network are black background, and Domains/Full Text are white background.

Let's make them consistent. I kind of like the contrast of a black background but happy to defer.

They're also stretched a bit weirdly - i.e. the "Full Text one" has a bit of skewing (as in screenshot below). What dimension do we need? Maybe we can create the images to fit the image instead of resizing them. I can chat with you on Slack to show you how to do neat Sublime Text themes or something.

![screen shot 2018-08-10 at 8 50 56 pm](https://user-images.githubusercontent.com/3834704/43986460-209edc76-9cdf-11e8-8541-e4a1e4181039.png)